{
  "modified": "2010-06-17T09:00:00Z",
  "published": "2010-05-27T19:00:00Z",
  "schema_version": "1.5.0",
  "id": "CVE-2010-1634",
  "aliases": [
    "PSF-2010-4"
  ],
  "details": "Multiple integer overflows in audioop.c in the audioop module in Python 2.6, 2.7, 3.1, and 3.2 allow context-dependent attackers to cause a denial of service (application crash) via a large fragment, as demonstrated by a call to audioop.lin2lin with a long string in the first argument, leading to a buffer overflow.  NOTE: this vulnerability exists because of an incorrect fix for CVE-2008-3143.5.",
  "affected": [
    {
      "package": {
        "ecosystem": "DHI",
        "name": "python",
        "purl": "pkg:dhi/python"
      },
      "ranges": [
        {
          "type": "SEMVER",
          "events": [
            {
              "fixed": "3.10.0-alpha1"
            },
            {
              "introduced": "3.10.0"
            },
            {},
            {}
          ],
          "repo": "https://github.com/python/cpython"
        }
      ]
    }
  ],
  "references": [
    {
      "type": "REPORT",
      "url": "https://bugs.python.org/issue8674"
    }
  ]
}